### PR TITLE
Doc format, add api-token section

### LINKED ISSUE #2971: Document usage of service account API tokens in REST API
### Is your feature request related to a problem? Please describe

Docs should mention how to use api tokens with rest api (`Authorization: Bearer $TOKEN`)

### PR DESCRIPTION
**Pull request by Firstyear**
Fixes #2971 - doc how the api token works, run doc/format/fix. 

Checklist

- [ x ] This PR contains no AI generated code
- [ x ] `cargo fmt` has been run
- [ ] `cargo clippy` has been run
- [ ] `cargo test` has been run and passes
- [ x ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: book/src/developers/designs/profile_display.md [hunk 1]
```diff
@@ -1,86 +1,105 @@
...
- - zoneinfo/timezone
- - locale/preferred language
- - other business related attributes: address, phone number, ...
+
+These consist of:
+
+- username
+- displayname
+- legal name
+- email address In the future:
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:22]:
```suggestion
 - email address 

In the future:

```

File: book/src/developers/designs/profile_display.md [hunk 2]
```diff
@@ -1,86 +1,105 @@
...
- - birthdate
+
+- addresses
+- phone numbers
+- personal emails
+- birthdate
 
 ### SSH public keys
+
 Ssh public key entries in kanidm consist of a:
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:52]:
```suggestion
Ssh public key entries in Kanidm consist of a:
```

File: book/src/developers/designs/profile_display.md [hunk 3]
```diff
@@ -1,86 +1,105 @@
...
- - label : practically the ID of the key in kanidm
- - value : the public key
 
-A user may want to change their laptop ssh key by updating the value while keeping the label the same.
-// TODO: Should a user be allowed to relabel their kanidm ssh keys ?
+- label : practically the ID of the key in kanidm
+- value : the public key
+
+A user may want to change their laptop ssh key by updating the value while keeping the label the
+same. // TODO: Should a user be allowed to relabel their kanidm ssh keys ?
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:58]: This TODO probably shouldn't be here 😄

File: book/src/developers/designs/profile_display.md [hunk 4]
```diff
@@ -1,86 +1,105 @@
...
 
+- label
+- value's key type (ECDSA, rsa, ect..) and may include:
+- value's comment, truncated to some max length
 
 #### Editing keys
-When editing keys users must be able to add keys, remove keys and update individual key values
-Each action will be committed immediately, thus proper prompts and icons indicating this must be shown (like a floppy disk save icon ?)
+
+When editing keys users must be able to add keys, remove keys and update individual key values Each
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:74]:
```suggestion
When editing keys users must be able to add keys, remove keys and update individual key values. Each
```

File: book/src/developers/designs/profile_display.md [hunk 5]
```diff
@@ -1,86 +1,105 @@
...
+Mostly a technical piece of info, should not be in direct view to avoid confusing users. Could be
+displayed in tree form.
 
 ### User profile HTML Structure
-To keep things oranised each category will be their own page with a subnavigation bar to navigate between them.
-Since htmx cannot (without extensions) swap new scripts into the <head> on swap during boosted navigation, we must do non-boosted navigation to our profile page OR enable some htmx extension library.
 
-The same htmx limitation means that all JS for every profile categories must be loaded on all profile categories.
-Because want to use htmx to swap out content on form submission or page navigation to represent the new state as this is more efficient than triggering the client to do a redirect.
+To keep things oranised each category will be their own page with a subnavigation bar to navigate
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:90]:
```suggestion
To keep things organised, each category will be their own page with a sub-navigation bar to navigate
```

File: book/src/developers/designs/profile_display.md [hunk 6]
```diff
@@ -100,19 +119,19 @@ And example would be
...
-<!-- ../static/profile.js -->
+// ../static/profile.js -->
 
 // Magically gets called on page load and swaps
 function onProfileSshKeysSwapped() {
-    // Do implementation things like attaching event listeners
+  // Do implementation things like attaching event listeners
 }
 
 indow.onload = function () {
```
Comment (yaleman) [inline book/src/developers/designs/profile_display.md:129]:
```suggestion
window.onload = function () {
```

File: book/src/domain_rename.md [hunk 7]
```diff
@@ -6,17 +6,17 @@ or other needs which may prompt this needing to be changed.
 
 > [!WARNING]
 >
-> This WILL break ALL u2f/webauthn tokens that have been enrolled, which MAY cause
-> accounts to be locked out and unrecoverable until further action is taken. DO NOT CHANGE the
-> domain name unless REQUIRED and have a plan on how to manage these issues.
+> This WILL break ALL u2f/webauthn tokens that have been enrolled, which MAY cause accounts to be
```
Comment (yaleman) [inline book/src/domain_rename.md:9]:
```suggestion
> This WILL break ALL U2F/WebAuthn tokens that have been enrolled, which MAY cause accounts to be
```

**Review by yaleman — CHANGES_REQUESTED** [review]
see comments 😄

File: book/src/domain_rename.md (hunk 7 above)
Comment (Firstyear) [reply book/src/domain_rename.md:9]: I have WebAuthn as a formatting so much.

**Review by yaleman — APPROVED** [review]

**Pull request merged**